blivet/formats/disklabel.py

```diff
@@ -324,6 +324,15 @@ def logicalPartitions(self):
         return logicals
 
     @property
+    def primaryPartitions(self):
+        try:
+            primaries = self.partedDisk.getPrimaryPartitions()
+        except Exception: # pylint: disable=broad-except
```

Comment:
I donť know, I basically took the logicalPartitions function and just changed logical to primary.